src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBtfBlocking.java

```diff
@@ -0,0 +1,164 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+public class TestAdsBtfBlocking extends TemplateNoFirstLoad {
+
+  private static final Dimension DESKTOP_PAGE_SIZE = new Dimension(1366, 768);
+  private static final Dimension TABLET_PAGE_SIZE = new Dimension(850, 600);
+  private static final Dimension MOBILE_SIZE = new Dimension(414, 736);
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "delayBtf",
+      groups = "AdsBtfBlockingOasis"
+  )
+  public void adsAtfDelayBtfOasis(String wikiName, String article, int delaySec, boolean isWgVarOn)
+      throws InterruptedException {
+    PageObjectLogging.log("$wgAdDriverDelayBelowTheFold", String.valueOf(isWgVarOn), true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "$wgAdDriverDelayBelowTheFold" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)